feature-libs/my-account/organization/assets/translations/en/permission.i18n.ts

```diff
@@ -1,28 +1,44 @@
 export const permission = {
-  header: 'Purchase limits',
-  disabled: '(disabled)',
+  header: 'All purchase limits ({{count}})',
+
   name: 'Name',
   code: 'Code',
+  active: 'Status',
   limit: 'Limit',
   orderApprovalPermissionType: 'Type',
   threshold: 'Threshold',
   periodRange: 'Period',
   currency: 'Currency',
   orgUnit: 'Parent Unit',
+  unit: 'Parent Unit',
+  actions: '',
+
+  details: {
+    title: 'Purchase limits details',
+    subtitle: 'Purchase limits: {{ item.code }}',
+  },
+  edit: {
+    title: 'Edit Purchase limit',
+    subtitle: 'Purchase limits : {{ item.code }}',
+  },
+  create: {
+    title: 'Create Purchase limit',
+    subtitle: '',
+  },
+
+  byName: 'Sort by name',
+  byUnitName: 'Sort by unit',
 
   messages: {
-    enabled:
-      'When you disable the purchase limit, the related data will be disabled as well. ',
-    disabled: 'You cannot edit a disabled purchase limit.',
     deactivate: 'Are you sure you want to disable this purchase limit?',
-    deactivateHeader: 'Disable Purchase Limit',
+    confirmEnabled: 'Purchase Limit enabled successfully',
+    confirmDisabled: 'Purchase Limit disabled successfully',
   },
 
   per: {
-    undefined: '',
-    MONTH: 'per Month',
-    YEAR: 'per Year',
```

Comment:
[organization:permission.per.YEAR] is missing now, why did you remove that?

Reply:
I've added the the year back. 
I undefined issue is solved by only rendering the `periodRange` for `isTimeSpanThreshold` type.

Reply:
Also fixed a bug to avoid overflow issue with the select.